packages/viem/src/__tests__/index-test.ts

```diff
@@ -34,35 +67,265 @@ describe("createApiKeyAccount", () => {
     expectedEthAddress = assertNonEmptyString(
       process.env.EXPECTED_ETH_ADDRESS,
       `process.env.EXPECTED_ETH_ADDRESS`
+    ) as Hex;
+
+    bannedToAddress = assertNonEmptyString(
+      process.env.BANNED_TO_ADDRESS,
+      `process.env.BANNED_TO_ADDRESS`
+    ) as Hex;
+
+    // create new client
+    const turnkeyClient = new TurnkeyClient(
+      {
+        baseUrl,
+      },
+      new ApiKeyStamper({
+        apiPublicKey,
+        apiPrivateKey,
+      })
     );
 
-    const turnkeyAccount = await createApiKeyAccount({
-      apiPublicKey,
-      apiPrivateKey,
-      baseUrl,
+    turnkeyAccount = await createAccount({
+      client: turnkeyClient,
       organizationId,
       privateKeyId,
     });
 
-    client = createWalletClient({
+    walletClient = createTestClient({
       account: turnkeyAccount,
-      chain: sepolia,
+      chain: foundry,
+      mode: "anvil",
       transport: http(),
+    })
+      .extend(publicActions)
+      .extend(walletActions);
+
+    chain = walletClient.chain;
+
+    await walletClient.setBalance({
+      address: expectedEthAddress,
+      value: parseEther("999999"),
     });
   });
 
-  // Create a .env file with credentials and modify this to "test(" to run tests
-  // TODO: make this work on CI
-  test.skip("basics", async () => {
-    expect(client.account?.address).toEqual(expectedEthAddress);
+  testCase("it signs transactions", async () => {
+    const request = await walletClient.prepareTransactionRequest({
+      account: turnkeyAccount,
+      chain,
+      to: "0x2Ad9eA1E677949a536A270CEC812D6e868C88108",
+      value: parseEther("1.0"),
+    });
+    const tx = await walletClient.signTransaction(request);
 
-    const signature = await client.signMessage({
-      message: "rno was here",
-      account: client.account!,
+    expect(tx).toMatch(/^0x/);
+  });
+
+  testCase("it sends transactions", async () => {
+    const transactionCount = await walletClient.getTransactionCount({
+      address: expectedEthAddress,
     });
-    expect(signature).toEqual(
-      "0x42ea50d0e54726f9ce0eabf2823381a89ee9131eb2095f431ced65087213a1b767c112bd892dd9221cd269364a9938bbe63c65eea4794abbb7b7d355eadeb9131c"
-    );
+
+    const txHash = await walletClient.sendTransaction({
+      account: turnkeyAccount,
+      to: "0x2Ad9eA1E677949a536A270CEC812D6e868C88108",
+      value: parseEther("1.0"),
+      chain,
+      nonce: transactionCount,
+      gas: 21000n,
+      maxFeePerGas: parseUnits("2", 9),
+      maxPriorityFeePerGas: parseUnits("2", 9),
+      type: "eip1559",
+    });
+
+    expect(txHash).toMatch(/^0x/);
```

Comment:
something that could be stronger than "txHash starts with 0x" could be: 1) fetch the tx details from hash, then 2) assert against the tx details ("to" should be the address we want, amount should be 1 ETH, etc)

Definitely nice-to-have though!

Reply:
i'll do these as a followup for both viem and ethers